### PR TITLE
New icons and copy changes for sidebar tutorial

### PR DESCRIPTION
**Pull request by lenazun**
Updates the icons for the sidebar tutorial and adds a line of copy for page notes.

@sheetaluk since you're the icon expert, can you help me with the CSS? I'm confused :person_frowning:

### CODE REVIEW

**Comment by lenazun** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;r4g6deHT&#x2F;319-add-a-page-note-item-to-the-how-to-get-started-sidebar-tutorial"> Add a page note item to the How To Get Started sidebar tutorial</a>

**Comment by robertknight** [comment]
@lenazun Is this still WIP or ready for review & merge now?

**Comment by lenazun** [comment]
@robertknight still WIP, working on it with @sheetaluk

File: h/static/styles/sidebar-tutorial.scss [hunk 1]
```diff
@@ -48,3 +50,8 @@
                       first list item (i.e. between the list and whatever's
                       above it). */
 }
+
+
+@include icons {
+  font-family: 'h';
+}
```
Comment (sheetaluk) [inline h/static/styles/sidebar-tutorial.scss]: We don't need this. Any class which begins with h-icon-, gets a font-family of h.
i.e [class^="h-icon-"], [class*=" h-icon-"]

File: h/static/styles/sidebar-tutorial.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import '../mixins/icons';
+
 .sidebar-tutorial__header {
```
Comment (sheetaluk) [inline h/static/styles/sidebar-tutorial.scss]: We don't need this either.

**Comment by sheetaluk** [comment]
Looks good. 2 comments: 

1. We don't need to +@import '../mixins/icons';
2. Also, we should probably squash the commits. I usually do:
git rebase -i HEAD~<n>
And follow whatever git tells me to do.

**Pull request merged**